matrix_registration/config.py

```diff
@@ -80,6 +83,14 @@ def load(self):
             try:
                 with open(self.data, 'r') as stream:
                     dictionary = yaml.load(stream, Loader=yaml.SafeLoader)
+                if not CONFIG_SCHEMA_PATH:
+                    sys.exit("Make sure you have "+CONFIG_SCHEMA_PATH)
```

Comment:
```suggestion
```
should be unnecessary?

Reply:
was meant for backwards compat but os.path.isfile(CONFIG_SCHEMA_PATH) would be better yes
-> could also remove it now idc

Reply:
well it will be packaged with the rest of the application. so if e.g. api.py is there, the schema should also be there. no need to test for that.

that said I just remembered that you need to add it to package data:
https://github.com/ZerataX/matrix-registration/blob/master/setup.py#L41
```
'config.schema.json',
```